src/base/projects/CloneButton.svelte

```diff
@@ -64,12 +62,16 @@
     text-overflow: ellipsis;
     white-space: nowrap;
     line-height: 1.4;
+    color: var(--color-caution-6);
+    background: linear-gradient(var(--color-caution-2), var(--color-caution-2)),
+      linear-gradient(var(--color-background), var(--color-background));
   }
   .clipboard {
     position: absolute;
     right: 0;
-    color: var(--color-yellow);
-    background: var(--color-yellow-background-solid);
+    color: var(--color-caution-6);
+    background: linear-gradient(var(--color-caution-2), var(--color-caution-2)),
```

Comment:
Why do we need two `linear-gradient` here and in `.clone-url`? Couldn't this be solved with `background: var(--color-caution-2);`

Reply:
It's a workaround to get the correct color because our color shades are transparencies. You get a different color when you layer multiple transparencies. We could also have flattened all our colors to solid values, but then we'd lose the flexibility to tweak them without introducing a transpile step (SCSS or the likes).

```diff
diff --git a/src/base/projects/CloneButton.svelte b/src/base/projects/CloneButton.svelte
index 107c398..0bd04f3 100644
--- a/src/base/projects/CloneButton.svelte
+++ b/src/base/projects/CloneButton.svelte
@@ -63,15 +63,13 @@
     white-space: nowrap;
     line-height: 1.4;
     color: var(--color-caution-6);
-    background: linear-gradient(var(--color-caution-2), var(--color-caution-2)),
-      linear-gradient(var(--color-background), var(--color-background));
+    background: var(--color-caution-2);
   }
   .clipboard {
     position: absolute;
     right: 0;
     color: var(--color-caution-6);
-    background: linear-gradient(var(--color-caution-2), var(--color-caution-2)),
-      linear-gradient(var(--color-background), var(--color-background));
+    background: var(--color-caution-2);
     visibility: hidden;
     width: 4rem;
     height: 2rem;
```

<img width="466" alt="Screenshot 2022-09-21 at 15 43 16" src="https://user-images.githubusercontent.com/158411/191520967-7bf8d8ed-a9f6-4536-b88e-41eefd5614cf.png">
vs with `linear-gradient` hack:
<img width="502" alt="Screenshot 2022-09-21 at 15 43 27" src="https://user-images.githubusercontent.com/158411/191520975-2f1a371d-406d-417d-bef4-f581220c30e3.png">

Reply:
Ok understood, thanks for the clarification 🙇